.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishTimeZoneExtractorConfiguration.cs

```diff
@@ -31,8 +33,14 @@ public class EnglishTimeZoneExtractorConfiguration : BaseOptionsConfiguration, I
 
         public EnglishTimeZoneExtractorConfiguration() : base(DateTimeOptions.None)
         {
+            CityStringMatcher.Build();
```

Comment:
This build cost should only be paid if EnablePreview is on. How are we controlling it? I didn't notice it in the PR.